data/istaustria/istaustria2018.csv

```diff
@@ -0,0 +1,16 @@
+institution,period,euro,doi,is_hybrid
+IST Austria,2018,2185.88,10.1371/journal.pgen.1007177,N
+IST Austria,2018,1975.11,10.1534/genetics.117.300513,Y
+IST Austria,2018,1'457.11,10.1371/journal.pone.0193049,N
+IST Austria,2018,692.76,10.1145/3197517.3201336,Y
+IST Austria,2018,5'514.42,10.1016/j.neuron.2018.02.024,Y
+IST Austria,2018,2'762.27,10.1073/pnas.1721760115,Y
+IST Austria,2018,2'499.00,10.1093/bioinformatics/bty340,Y
+IST Austria,2018,571.79,10.24963/ijcai.2018/652,N
+IST Austria,2018,864.88,10.1017/fms.2018.7,N
+IST Austria,2018,2'226.96,10.1016/j.neuron.2018.02.024,Y

```

Comment:
This one forms a DOI duplicate with line 6 (10.1016/j.neuron.2018.02.024) and the costs differ. Could you make clear which one is correct?